### PR TITLE
Add description on `admin pause ddl jobs` and `admin resume ddl jobs`

### PR DESCRIPTION
**Pull request by dhysum**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add description about the new admin command:
- admin pause ddl jobs
- admin resume ddl jobs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
    - https://github.com/pingcap/tidb/pull/43081
    - https://github.com/pingcap/tidb/pull/43159
- Other reference link(s):
    - https://github.com/pingcap/tidb/issues/18015 
    - https://github.com/pingcap/tidb/issues/40041

Related issue number: 
- https://github.com/pingcap/tidb/issues/39751 
- https://github.com/pingcap/tidb/pull/43061

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
cc @Benjamin2037

File: ddl-introduction.md
Comment (ran-huang) [inline ddl-introduction.md:1]: `ADMIN PAUSE` 和 `ADMIN RESUME` 要添加文档吗？类似这一篇 [ADMIN SHOW DDL [JOBS|JOB QUERIES]](https://docs.pingcap.com/zh/tidb/stable/sql-statement-admin-show-ddl)

File: ddl-introduction.md [hunk 1]
```diff
@@ -178,6 +178,15 @@ absent -> delete only -> write only -> write reorg -> public
 
     取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。在暂停之后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。
+
+    不在执行中或等待中的任务，无法暂停，会在 RESULT 列看到 `Job 3 can't be paused now`。
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行之后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。
```
Comment (ran-huang) [inline ddl-introduction.md:185]:
```suggestion
- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。
```

File: ddl-introduction.md [hunk 2]
```diff
@@ -178,6 +178,15 @@ absent -> delete only -> write only -> write reorg -> public
...
     取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。
+
+    不在执行中或等待中的任务，无法暂停，会在 RESULT 列看到 `Job 3 can't be paused now`。
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。
+
+    非暂停状态的任务无法恢复运行，会在 RESULT 列看到 `Job 3 can't be resumed`
+
```
Comment (ran-huang) [inline ddl-introduction.md:188]:
```suggestion
```

**Review by ran-huang — APPROVED** [review]

File: ddl-introduction.md [hunk 3]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
 
     取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。
+
+    不在执行中或等待中的任务，无法暂停，会在 RESULT 列看到 `Job 3 can't be paused now`。
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。
+
+    非暂停状态的任务无法恢复运行，会在 RESULT 列看到 `Job 3 can't be resumed`
```
Comment (ran-huang) [inline ddl-introduction.md:187]:
```suggestion
    非暂停状态的任务无法恢复运行，会在 RESULT 列看到 `Job 3 can't be resumed`。
```

File: ddl-introduction.md [hunk 4]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
 
     取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。
```
Comment (ran-huang) [inline ddl-introduction.md:181]:
```suggestion
- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。（实验特性）
```

File: ddl-introduction.md [hunk 5]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
...
+
+    不在执行中或等待中的任务，无法暂停，会在 RESULT 列看到 `Job 3 can't be paused now`。
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。
```
Comment (ran-huang) [inline ddl-introduction.md:185]:
```suggestion
- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。（实验特性）
```

**Review by benmaoer — APPROVED** [review]
LGTM

File: ddl-introduction.md [hunk 6]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
 
     取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。
+
+    不在执行中或等待中的任务，无法暂停，会在 RESULT 列看到 `Job 3 can't be paused now`。
```
Comment (qiancai) [inline ddl-introduction.md:183]:
```suggestion
    不在执行中或仍在等待中的 DDL 任务，无法暂停，会在 `RESULT` 列看到 `Job 3 can't be paused now`。
```

File: ddl-introduction.md [hunk 7]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
 
     取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
```
Comment (qiancai) [inline ddl-introduction.md:179]:
```suggestion
    取消一个已经执行完成的 DDL 任务会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
```

**Review by qiancai — APPROVED** [review]
other LGTM

File: ddl-introduction.md [hunk 8]
```diff
@@ -176,7 +176,16 @@ absent -> delete only -> write only -> write reorg -> public
...
-    取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
+    取消一个已经执行完成的 DDL 任务会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
+
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。（实验特性）
+
+    只有处于执行中或仍在等待中的 DDL 任务可以暂停，否则会在 `RESULT` 列看到 `Job 3 can't be paused now`。
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。（实验特性）
+
+    你只能对暂停状态的 DDL 任务进行恢复操作，否则会在 `RESULT` 列看到 `Job 3 can't be resumed`
```
Comment (qiancai) [inline ddl-introduction.md:187]:
```suggestion
    你只能对暂停状态的 DDL 任务进行恢复操作，否则会在 `RESULT` 列看到 `Job 3 can't be resumed`。
```

File: ddl-introduction.md [hunk 9]
```diff
@@ -176,7 +176,16 @@ absent -> delete only -> write only -> write reorg -> public
 
 - `ADMIN CANCEL DDL JOBS job_id [, job_id]`：用于取消已经提交但未执行完成的 DDL 任务。取消完成后，执行 DDL 任务的 SQL 语句会返回 `ERROR 8214 (HY000): Cancelled DDL job` 错误。
 
-    取消一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
+    取消一个已经执行完成的 DDL 任务会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
+
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。（实验特性）
+
+    只有处于执行中或仍在等待中的 DDL 任务可以暂停，否则会在 `RESULT` 列看到 `Job 3 can't be paused now`。
```
Comment (qiancai) [inline ddl-introduction.md:183]: 再确认下，等待中的 DDL 任务可以暂停吗

File: ddl-introduction.md [hunk 10]
```diff
@@ -176,7 +176,16 @@ absent -> delete only -> write only -> write reorg -> public
...
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。（实验特性）
+
+    你只能对暂停状态的 DDL 任务进行恢复操作，否则会在 `RESULT` 列看到 `Job 3 can't be resumed`。
+
```
Comment (ran-huang) [inline ddl-introduction.md:188]:
```suggestion
```

**Pull request merged**